### PR TITLE
Update the game over message

### PR DESCRIPTION
**Pull request by Chan-Jing-Hui**
Update the game over message so people know how to play again!

### CODE REVIEW

**Review by Chan-Jing-Hui — COMMENTED** [review]
Good Change!

**Review by Chan-Jing-Hui — COMMENTED** [review]
Review 2 (0)

File: index.html
Comment (Sludge44) [inline index.html:1]: mklfg

**Comment by Chan-Jing-Hui** [comment]
hthrhljkljtjkl;

**Comment by Chan-Jing-Hui** [comment]
WROORKOKROWKWORKWORKWROKWROWKRWORK

**Pull request merged**